airflow_prometheus_exporter/prometheus_exporter.py

```diff
@@ -372,6 +378,32 @@ def get_num_queued_tasks():
             .count()
         )
 
+def insert_latest_successful_run(insert_dict):
+    for key, execution_date in insert_dict.items():
+        dag_id, task_id = key
+        session.add(LatestSuccessfulRun(
```

Comment:
this looks like it is referring to the session here
https://github.com/Openmail/airflow-prometheus-exporter/pull/28/files#diff-e567aea89df0f7c267efa5eeec1425e14a0f6052b3767a9be7c77b2948e890f6R44
but this session would be used outside of the context

Reply:
i think you should be defining the session like the other functions
```    with session_scope(Session) as session:```

Reply:
Yea, that should be passed from that functions (dag / task). It worked accidentally.